src/collection/components/CollectionFragment.tsx

```diff
@@ -296,7 +297,14 @@ const CollectionFragment: FunctionComponent<CollectionFragmentProps> = ({
 					updateCollection={updateCollection}
 					reorderFragments={reorderFragments}
 				/>
-				<CutFragmentModal isOpen={isCutModalOpen} setIsOpen={setIsCutModalOpen} />
+				{itemMetaData && (
+					<CutFragmentModal
+						isOpen={isCutModalOpen}
+						setIsOpen={setIsCutModalOpen}
+						itemMetaData={itemMetaData}
+						externalId={fragment.external_id}
```

Comment:
hier is het mss mooier om toch external id mee te selecteren wnr je de itemMetaData ophaalt, dan kan je deze component instantieren door enkel itemMetaData mee te geven zonder de extra param externalId